ebos/eclgenericwriter_impl.hh

```diff
@@ -263,33 +264,44 @@ writeInit(const std::function<unsigned int(unsigned int)>& map)
 }
 
 template<class Grid, class EquilGrid, class GridView, class ElementMapper, class Scalar>
-data::Solution EclGenericWriter<Grid,EquilGrid,GridView,ElementMapper,Scalar>::
+data::Solution
+EclGenericWriter<Grid,EquilGrid,GridView,ElementMapper,Scalar>::
 computeTrans_(const std::unordered_map<int,int>& cartesianToActive,
               const std::function<unsigned int(unsigned int)>& map) const
 {
     const auto& cartMapper = *equilCartMapper_;
     const auto& cartDims = cartMapper.cartesianDimensions();
-    const int globalSize = cartDims[0] * cartDims[1] * cartDims[2];
 
     auto tranx = data::CellData {
         UnitSystem::measure::transmissibility,
-        std::vector<double>(globalSize, 0.0),
+        std::vector<double>(cartDims[0] * cartDims[1] * cartDims[2], 0.0),
         data::TargetType::INIT
     };
 
     auto trany = tranx;
     auto tranz = tranx;
 
-    using GlobalGridView = typename EquilGrid::LeafGridView;
-    const GlobalGridView& globalGridView = equilGrid_->leafGridView();
-    using GlobElementMapper = Dune::MultipleCodimMultipleGeomTypeMapper<GlobalGridView>;
-    GlobElementMapper globalElemMapper(globalGridView, Dune::mcmgElementLayout());
+    const auto& globalGridView = this->equilGrid_->leafGridView();
+    const auto globalElemMapper = Dune::MultipleCodimMultipleGeomTypeMapper
+        <typename EquilGrid::LeafGridView>
+    {
+        globalGridView, Dune::mcmgElementLayout()
```

Comment:
> Would prefer formatting making it clear that this is an initialization, and since you end up using the full name of the type anyway, the original style is preferable to me.

Fair enough.  I was trying to reduce the number of single-purpose names, but I can revert to the original&ndash;except for keeping the `globalElemMapper` be `const`.  It was mutable in the original code.

Reply:
Const is always good!

Reply:
I've pushed an update to restore the original names.